Update docs for new external ID retrieval workflow (#938)
astronomer/docs: astro/install-aws.md

:::

> ## Step 2: Retrieve an external ID from Astro

Author: ```suggestion
## Step 2: Retrieve an external ID from the Cloud UI
```

> In the Cloud UI, open the **Settings** tab and copy the value in **AWS EXTERNAL ID**. This external ID is a unique identifier that Astro uses to connect to your AWS account. Save the external ID as a secret or in another secure format. See [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).

Reviewer: ```suggestion
In the Cloud UI, open the **Settings** tab and copy the value in **AWS EXTERNAL ID**. This external ID is a unique identifier that Astro uses to connect to your AWS account. Save the external ID as a secret or in another secure format. See [How to use an external ID when granting access to your AWS resources to a third party](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).

You must be an Organization Owner to view the external ID. If you are not an Organization Owner, the field will not appear in the Cloud UI.
```

:::

## Step 4: Provide setup information to Astronomer

> For the AWS account you created as a prerequisite, provide Astronomer with:

Reviewer 2: ```suggestion
After creating the AWS account, provide Astronomer support with the following information:
```

- Your AWS Account ID.
- Your preferred Astro cluster name.
- The AWS region that you want to host your cluster in.
- Your preferred node instance type.
- Your preferred maximum node count.

> If not specified, Astronomer will create a cluster with two `m5.xlarge` nodes in `us-east-1` and a maximum node count of 20 by default. For information on all supported regions and configurations, see [AWS resource reference](resource-reference-aws.md).

Reviewer 2: ```suggestion
If you don’t provide the necessary information, Astronomer support creates a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information about supported AWS regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
```

Author: Same comment as before

Reviewer 2: @jwitz see my previous response.

> From here, our team will provision an Astro cluster according to the specifications you provided.

Reviewer 2: ```suggestion
Astronomer support provisions an Astro cluster using the setup information you provided.
```

Reviewer 2: Is this statement necessary?

Author: I think the title of the next step implies this, good call we should remove

> ## Step 5: Let Astronomer complete the install

Reviewer 2: ```suggestion
## Step 5: Astronomer completes the install
```

Author: Maybe "Wait for Astronomer to complete the install"? I like having the step start with a verb

> Contact Astronomer after you've created the cross-account IAM role for Astro. From there, Astronomer will finish creating the cluster in your AWS account.

Reviewer 2: ```suggestion
After you've created the cross-account IAM role for Astro, contact Astronomer support. Astronomer support will finish creating the cluster in your AWS account.
```

:::

## Step 4: Provide setup information to Astronomer

After creating the AWS account, provide Astronomer support with the following information:

- Your AWS Account ID.
- Your preferred Astro cluster name.
- The AWS region that you want to host your cluster in.
- Your preferred node instance type.
- Your preferred maximum node count.

> If not specified, Astronomer will create a cluster with two `m5.xlarge` nodes in `us-east-1` and a maximum node count of 20 by default. For information on all supported regions and configurations, see [AWS resource reference](resource-reference-aws.md).

Reviewer: ```suggestion
If you do not specify configuration preferences, Astronomer will create a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
```

## Step 5: Wait for Astronomer to complete the install

> After you've created the cross-account IAM role for Astro, contact Astronomer support. Astronomer support will finish creating the cluster in your AWS account.

Reviewer: ```suggestion
After you've created the cross-account IAM role for Astro, contact [Astronomer support](https://support.astronomer.io). Astronomer support will finish creating the cluster in your AWS account.
```
